src/main/kotlin/app/revanced/utils/adb/Adb.kt

```diff
@@ -15,6 +15,9 @@ internal class Adb(
     private val logging: Boolean = true
 ) {
     private val device: JadbDevice
+    companion object {
+        var SuperSU: Boolean = true
```

Comment:
variables start with a lower case.

Reply:
I commited the review by Lucas as it made more sense in the context whilst following the naming style.